app/access_policies/application_user_access_policy.rb

```diff
@@ -5,8 +5,8 @@ def self.action_allowed?(action, requestor, application_user)
     # Human users are not allowed
     return false if requestor.is_human?
 
-    # Apps can only call search, updates and updated
-    return [:search, :updates, :updated].include?(action)
+    # Apps can only call read, search, updates and updated
+    return [:read, :search, :updates, :updated].include?(action)
```

Comment:
Updates and updated only return users for the current app (and they  
return multiple users) so the check does not apply.

Quoting Nathan Stitt <notifications@github.com>:

>> @@ -5,8 +5,8 @@ def self.action_allowed?(action, requestor,  
>> application_user)
>>      # Human users are not allowed
>>      return false if requestor.is_human?
>>
>> -    # Apps can only call search, updates and updated
>> -    return [:search, :updates, :updated].include?(action)
>> +    # Apps can only call read, search, updates and updated
>> +    return [:read, :search, :updates, :updated].include?(action)
>
> Yep, I've implemented that check now.  Should the other `:updates`  
> and `:updated` perform the same check though?  That's probably  
> outside the scope of this issue, but we might want to file another  
> one if that's the case.
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/openstax/accounts/pull/137/files#r26236223